operations/messaging-and-math/how-to-guides-for-m-and-m/how-to-create-release-announcements.md

```diff
@@ -21,6 +21,7 @@ description: 50% Draft
   * Marketing reviews
 * Head of PM reviews 
 * CEO review
+   - Use hashtags in marketing channel (e.g. #v5.20-blog) to make finding the blog draft easier, and post the blog draft in the marketing channel 24 hours prior to the meeting (and at-mention meeting attendees)
```

Comment:
Who uses hashtags? The CEO?

Should we use the same hashtag we'll use for tweets, e.g. `#mm520`